tables.md

```diff
@@ -0,0 +1,204 @@
+# Tables
+
+When writing your JSON-LD file to accompany your csv, you can write it in a way to include multiple csvs.
+
+Below is an extract of two csv that will be described in the JSON-LD.
+
+The first one is called football.csv.
+
+| period_type | period_code | variable_name     | observation | measure | unit       |
+| ----------- | ----------- | ----------------- | ----------- | ------- | ---------- |
+| Year        | 2023        | Manchester United | 6           | billion | U.S dollar |
+| Year        | 2023        | Liverpool         | 5.3         | billion | U.S dollar |
+| Year        | 2023        | Manchester City   | 5           | billion | U.S dollar |
+| Year        | 2023        | Chelsea           | 3.1         | billion | U.S dollar |
+| Year        | 2023        | Tottenham Hotspur | 2.8         | billion | U.S dollar |
+
+The second one is american_football.csv.
+
+| period_type | period_code | variable_name        | observation | measure | unit       |
+| ----------- | ----------- | -------------------- | ----------- | ------- | ---------- |
+| Year        | 2023        | Dallas Cowboys       | 9           | billion | U.S dollar |
+| Year        | 2023        | New England Patriots | 7           | billion | U.S dollar |
+| Year        | 2023        | Los Angeles Rams     | 6.9         | billion | U.S dollar |
+| Year        | 2023        | New York Giants      | 6.8         | billion | U.S dollar |
+| Year        | 2023        | Chicago Bears        | 6.3         | billion | U.S dollar |
+
+There are additional descriptors for human readability that you can add to your column sections. These include:
+
+- `suppressOutput`  If true, suppresses any output that would be generated when converting cells in this column. The value of this property becomes the suppress output annotation for the described column. The default is false.(from w.3/org-tabular-metadata)
```

Comment:
Lots of typos here. The point is how to encode for manchine readability, but keeping the human readability within the csv.

For humans, we recommend keeping to a pattern of dimension_code, dimension_label in the csv; but when encoding for machines, we recommend suppressing the dimension_label if the dimension_code resolves to an existing resource with the label present, you ensure machine readbaility by doing x, y, or z.

Reply:
I have placed your wording in the area. I think it might be best to provide an example to make it easier.